client/src/session.rs

```diff
@@ -1721,10 +1733,20 @@ mod tests {
 
         // Function should finish in timeout and return error.
         assert!(
-            timeout(Duration::from_millis(4500), layer1.layer.session(layer2.id))
+            timeout(os_specific_timeout(), layer1.layer.session(layer2.id))
```

Comment:
This timeout failed few times both on GH Windows VMs and on my local Windows machine.
I introduced OS specific timeout, but this PR could just move test closer to failure threshold.

Reply:
It is weird. Locally on Windows these tests were sometimes failing with:
```
---- session::tests::test_session_layer_p2p_timeout_handshake stdout ----
deadline has elapsed
thread 'session::tests::test_session_layer_p2p_timeout_handshake' panicked at 'called `Result::unwrap()` on an `Err` value: Elapsed(())', client\src\session.rs:1742:18
```
Meanwhile on CI it could sometimes fail on Windows with:
```
---- session::tests::test_session_layer_close_relayed_routing stdout ----
[2023-08-10 15:19:25.868 ERROR ya_relay_server::server] Packet dispatch failed (request=Some(33), from=127.0.0.1:59034). BadRequest: No Public Endpoints.
thread 'session::tests::test_session_layer_close_relayed_routing' panicked at 'called `Result::unwrap()` on an `Err` value: Generic("Network error: Request failed: Request failed with code 400")', client/src/session.rs:1655:62
```
Maybe these tests should use `serial_test`?